Makefile

```diff
@@ -150,6 +150,7 @@ entropy:
 	bash -f .ci/install_bats.sh
 	cd integration/entropy && \
 	bats entropy_test.bats
+	bats entropy_time.bats
```

Comment:
A niggle... is this right? I'm not sure the `cd` carries over to the 2nd line does it?
We seem to have a mix through the `Makefile` where some `bats` tests use the `cd && bats` idiom, and others use the `bats fullpath` idiom. I prefer the second (the fullpath) one?
Maybe make both these entropy bats lines `bats <fullpath to batsfile>` style?